File: app/src/main/java/com/tda/app/data/model/ProductResponse.java
package com.tda.app.data.model;

import java.sql.Timestamp;
import java.util.List;

import lombok.NoArgsConstructor;


@NoArgsConstructor
public class ProductResponse {
    public long productId;
    public String productCode;
    public String productName;
    public String description;
    public int quantity;
    public int status;
    public double price;
    public double promotionPrice;
    public Timestamp createDate;
    public Timestamp updateDate;
    public boolean isDeleted;
    public long categoryId;
    public List<ProductType> productType;
    public List<String> images_file;


}

File: app/src/main/java/com/tda/app/model/response/ProductResponse.kt
package com.tda.app.model.response

import java.sql.Timestamp


data class ProductResponse(
    val productId: Long,
    val productCode: String,
    val productName: String,
    val description: String,
    val quantity: Int,
    val status: Int,
    val price: Double,
    val promotionPrice: Double,
    val createDate: Timestamp,
    val updateDate: Timestamp,
    val isDeleted: Boolean,
    val categoryId: Long,
    val images_file: List<String>
)